### PR TITLE
Add height theme prop to ShopLogo

### LINKED ISSUE #403: Can't set image height on ShopLogo
Type: **major**

**Describe the bug**
There is currently no way to control the height of `ShopLogo`'s `img` component. Adding a `className` to the container (which is not advised) doesn't allow to control the height of the logo.

**To Reproduce**
1. Use `ShopLogo` to render an image and not the shop's name
2. See that it renders without any constraints in dimensions
3. Try to restrain it to a given height
4. Fail miserably, as my example shows:

screen shot 2019-03-03 at 17 08 03

**Expected behavior**
Developers should have a way to set the logo's maximum height with a prop.

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome
 - Version 72

### PR DESCRIPTION
**Pull request by loan-laux**
Resolves #403
Impact: **minor**  
Type: **feature**

## Component
There was no way to set `ShopLogo`'s `img` height through theming, which is a bit of a pain when using vector images as they appear huge by default.

## Screenshots
Here's a test with `
screen shot 2019-03-03 at 19 36 26
screen shot 2019-03-03 at 19 36 43

## Breaking changes
None.

## Testing
1. Change the value of `ShopLogo.height` in `src/theme/defaultComponentTheme.js`.
2. See that the the placeholder image is now at your desired height on `http://localhost:4040/#!/ShopLogo`.

### CODE REVIEW

**Comment by rc-publisher** [comment]
🚀 Preview deployed

Built with commit b362329e52bdf2d6457f4aaea749e0af96e97444

https://deploy-preview-404--stoic-hodgkin-c0179e.netlify.com

**Comment by rc-publisher** [comment]
🚀 Preview deployed

Built with commit 4cde2e9681f1c098bbc34f428744a00a55376357

https://deploy-preview-404--stoic-hodgkin-c0179e.netlify.com

File: package/src/components/ShopLogo/v1/ShopLogo.js [hunk 1]
```diff
@@ -26,13 +30,13 @@ export default class ShopLogo extends Component {
   }
 
   render() {
-    const { className, shopLogoUrl, shopName } = this.props;
+    const { className, height, shopLogoUrl, shopName } = this.props;
```
Comment (aldeed) [inline package/src/components/ShopLogo/v1/ShopLogo.js]: Remove `height` from here

Comment (loan-laux) [reply package/src/components/ShopLogo/v1/ShopLogo.js]: My bad, that's a leftover from when I first wanted to add it as an actual prop. Then I realized that a theme prop would be more appropriate.

**Comment by loan-laux** [comment]
@aldeed Should be good now, thanks for the heads up on that unused const.

**Comment by aldeed** [comment]
Sorry @loan-laux one more thing I didn't notice first pass. You need to run `yarn test -u` to update the Jest snapshot that has changed as a result of your change, and then commit the updated snapshot file. That should get CI tests passing.

**Comment by loan-laux** [comment]
Sorry I had a busy week @aldeed. Tried to run `yarn test -u` with `node_modules` properly installed, but I get the following error on almost all the tests:

```
Cannot find module 'accounting-js' from 'formatMoney.js'

    However, Jest was able to find:
    	'./formatMoney.js'
```

Have you seen that before?

**Comment by aldeed** [comment]
@loan-laux There are two `package.json` that need to be installed for this project.

```
yarn install
cd package
yarn install
cd ..
yarn test -u
```

**Comment by loan-laux** [comment]
Sorry about that @aldeed. Updated the snapshot, all tests are green now!

**Comment by loan-laux** [comment]
Hmm, Snyk is in the red due to authentication as usual. But other than that, seems fine.

**Pull request merged**